inst/ms/aggreCAT.qmd

```diff
@@ -536,27 +565,26 @@ an exemplar aggregation method from each mathematical aggregator
 #| prompt: true
 set.seed(1234)
 focal_claims <- data_ratings %>% 
-  filter(paper_id %in% c("24", "138", "186", "108"))
+  dplyr::filter(paper_id %in% c("24", "138", "186", "108"))
 # select 5 users to highlight in focal claim demonstration
 focal_users <- focal_claims %>% 
-  distinct(user_name) %>% 
-  sample_n(5) %>% 
-  mutate(participant_name = paste("participant", rep(1:n())))
+  dplyr::distinct(user_name) %>% 
+  dplyr::slice_sample(n=5) %>% 
+  dplyr::mutate(participant_name = paste("participant", rep(1:n())))
```

Comment:
Is this step worth it over the existing names?

Reply:
- [ ] Potentially not, @eggouldo to drop